### PR TITLE
Remove tutorial

### PR DESCRIPTION
**Pull request by vitvakatu**

### CODE REVIEW

**Review by stanislav-tkach — DISMISSED** [review]

**Review by alekseysidorov — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Just add the following line to the `Cargo.toml`:
 exonum-testkit = "0.1.0"
 ```
 
-[Read more...][tutorial]
+[Tutorial][tutorial]
```
Comment (slowli) [inline README.md]: Nitpick: I wouldn't call that particular page a tutorial. There *is* a separate tutorial (exonum/exonum-doc#143) showing how to use testkit on the example of the demo cryptocurrency. So, I would change to something like "See Exonum documentation for details". A link to the actual tutorial could be added later when it's merged and published (hopefully, soon).

**Review by slowli — DISMISSED** [review]
LGTM as of 9492db66b9d76bd02932c1611fba6210f98ef17d, with a nitpick.

**Comment by vitvakatu** [comment]
@DarkEld3r what about merging this before https://github.com/exonum/exonum/pull/448 ?

**Comment by matklad** [comment]
@vitvakatu blocking large PR on a small PR sounds like priority inversion :) 

I am all for merging #448 as soon as tests pass, even everything is not perfect, and than fix stuff in follow-up PRs :-)

**Pull request closed without merging**